shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -3,13 +3,19 @@
 // found in the LICENSE file.
 
 #import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterEngine.h"
+#include "FlutterCodecs.h"
```

Comment:
This should move down to the other flutter headers that aren't directly associated with `FlutterEngine` and prefix with the full include path.

Reply:
Damn you VSCode! I hate when it adds headers for me. I've turned it off in every place I can find, but clearly I've missed a spot.
It's not even needed (anymore: it probably was needed at one point in my explorations).